src/cmd_line.rs

```diff
@@ -101,6 +109,18 @@ pub fn handle_command_line_arguments() -> Result<(), String> {
                 .takes_value(true)
                 .last(true)
                 .help("Specify Arguments to pass down to neovim"),
+        )
+        .arg(
+            Arg::with_name("wayland_app_id")
+                .long("appid")
```

Comment:
```suggestion
                .long("wayland-app-id")
```

I'd suggest using this format, and dropping short version. `-a` is too ambiguous.